test/e2e/filters.spec.js

```diff
@@ -30,9 +30,10 @@ test('can unset filters using filter tags', async ({ page }) => {
   await expect(cc0Tag).toHaveCount(1)
   page.on('requestfinished', (request) => {
     const url = request.url()
-    const baseUrl = 'https://api.openverse.engineering/v1/images'
+    // Only check the URL for an image search query `?`, not thumbs or related requests
+    const baseUrl = 'https://api.openverse.engineering/v1/images/?'
```

Comment:
It's best to not have the `?` be a part of the base URL and rather keep it with the query params.

Reply:
Here, I'm trying to test the URL of the search query. There are 3 requests going to the API: one for thumbs, one for related, and one for the actual search. All three start with `https://api.openverse.engineering/v1/images`, so to be sure that we select only the search query url, I added the question mark here. I'm sure there are more elegant solutions, but I can't currently find them, @dhruvkb . 
Without the question mark, the tests can fail because they test the wrong request URL, and not because they are actually failing.

Reply:
In that case, I will rescind my comment and approve the PR. Thanks for fixing the broken tests!